.github/workflows/autotag.yaml

```
@@ -7,7 +7,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@main
```

```suggestion
      - uses: actions/checkout@v2
```

pinning to a major version would be even more future proof